week-6/learning-outcomes.md

```diff
@@ -18,36 +19,36 @@
 
 ### Promises
 
-- [ ] We can create a promise from scratch
+- [ ] **We can create our own promises**
 - [ ] We can wrap a function that takes a callback in a promise
 
 ## Design
 
 ### Sign-in form
 
-- [ ] We can design a usable sign in form
+- [ ] **We can design a usable sign in form**
 - [ ] We can create a sign in form with semantic accessible HTML
 - [ ] We can help users enter the right information with HTML5 validation
 
 ## Testing
 
 ### Test coverage
 
-- [ ] We can measure the test coverage of our code
+- [ ] **We can measure the test coverage of our code**
 - [ ] We can identify where our tests are lacking and fix this
 
 ### Static analysis
 
-- [ ] We can configure ESLint to analyse our code
+- [ ] **We can configure ESLint to analyse our code**
 - [ ] We can describe how a linter helps us write code
```

Comment:
How would we test for this? Is something like *"We can demonstrate a failed git merge due to inconsistencies in formatting"* too silly?

Reply:
The idea of this being in the "Testing" section was showing how ESLint can prevent actual runtime bugs (e.g. by catching an undefined variable). It's as close as you can get to typechecking vanilla JS 😆

I would say Prettier is a much better choice for _formatting_ at this point.

If we wanted a more specific outcome we could use "We can configure ESLint to catch common JS errors"?